rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -716,4 +720,20 @@ public long getGasEstimationCap() {
     public DbKind databaseKind() {
         return DbKind.ofName(configFromFiles.getString(KeyValueDataSource.KEYVALUE_DATASOURCE_PROP_NAME));
     }
+
+    public long getGasCap() {
+        return configFromFiles.getLong(PROPERTY_RPC_GAS_CAP);
+    }
+
+    public int getRpcTimeout() {
+        return configFromFiles.getInt(PROPERTY_RPC_TIMEOUT);
+    }
+
+    public int getEvmTimeout() {
+        return configFromFiles.getInt(PROPERTY_RPC_EVM_TIMEOUT);
+    }
+
+    public int getGetLogsTimeout() {
```

Comment:
This and `getEvmTimeout` methods and their constants are not needed, no? They are not used and I see we define timeouts at module+method level in properties.

Reply:
Done removed